### DISCUSSION TITLE
How do I display Bible verses with number in the front?

### DISCUSSION THREAD

**Discussion by hahamigo** [tim-hub/obsidian-bible-reference#143] [Q&A]
I have been trying the following:
`--1John 1:1`

However, it never displays the verses I need, like the others. How do I do this?

**Comment by tim-hub** [comment]
Sorry, this is a bug, from upstream lib, in progress to fix it now. 


https://github.com/tim-hub/obsidian-bible-reference/issues/139

**Reply by hungry4grace** [reply]
Please upgrade the importance of this bug. Thanks.
